Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.py

```diff
@@ -1179,7 +1179,8 @@ def device_list_automatic_pagination(request_by_device_type: Callable, client, c
     continue_pagination = True  # This will decide if we should continue requesting from the API or that we should stop
     while continue_pagination:
         query_params['maxResults'] = results_limit if results_limit <= MAX_PAGE_SIZE else MAX_PAGE_SIZE
-        query_params['pageToken'] = next_page_token
+        if next_page_token:
```

Comment:
and what happens if the next_page_token is empty?

Reply:
We get an error: **Invalid Input**, since the **pageToken** argument expects a correct page token, and not an empty string

Reply:
yes, but in our code - do we handle this situation? should we raise an exception in that case?

Reply:
The thing is that the user does not have to enter a page token, therefore the argument, **page_token**, can be an empty string, which is its default value, so technically, it is not a problem if the argument carries an empty string, we just need to make sure that an empty **page_token** does not get sent with the API call.